packages/core/test/shared/ws.ts

```diff
@@ -1,7 +1,7 @@
 import { IRelayer } from "@walletconnect/types";
 
 export async function disconnectSocket(relayer: IRelayer) {
-  if (relayer.connected) {
+  if (relayer && relayer.connected) {
```

Comment:
is this required for the fallback relay url test only? Asking if we need to set it in sign-client tests as well

Reply:
Not required as such for the tests, but while I was tinkering with the logic I noticed that this would end up throwing on every test if `relayer.init` failed, thus obscuring what the actual throw/issue was.